backend/src/main/java/gov/cdc/usds/simplereport/service/model/SmsDeliveryResult.java

```diff
@@ -0,0 +1,26 @@
+package gov.cdc.usds.simplereport.service.model;
+
+public class SmsDeliveryResult {
+  private String telephone;
+  private String messageId;
+  private Boolean deliverySuccess;
```

Comment:
**nitpick (non-blocking):** I think this can just be the `boolean` primitive type

Reply:
Good call, changed in 97cfa11.